ci/tests/general/general_vdisk.py

```diff
@@ -163,9 +166,10 @@ def disconnect_volume(loop_device, root_client=None):
 
         try:
             if loop_device is not None:
-                root_client.run("""umount /mnt/{0}; rmdir /mnt/{0}""".format(loop_device))
+                root_client.run('umount /mnt/{0}'.format(loop_device), allow_nonzero=True, allow_insecure=True)
+                root_client.run('rmdir /mnt/{0}'.format(loop_device), allow_nonzero=True, allow_insecure=True)
             else:
-                root_client.run('rmdir /mnt/{0}'.format(loop_device))
+                root_client.run('rmdir /mnt/{0}'.format(loop_device), allow_nonzero=True, allow_insecure=True)
```

Comment:
change to root_client.run(['rmdir', '/mnt/{0}'.format(loop_device)], allow_nonzero=True)